### ISSUE TITLE
Числа в генеримых текстах

### ISSUE THREAD

**Issue by Elevator89** [Ludeon/RimWorld-ru#1298]
Внимание на коммит b54199d2d5f5d3b325291ca286ad1f48a449a7e3
Когда разработчики дадут возможность пользоваться символами в генеримых текстах, надо заменить всё, что в входило этот коммит, на правильные функции

**Comment by MarisKori** [comment]
Речь про эти числа?
![image](https://user-images.githubusercontent.com/18173966/127679518-1c49fcd8-62ad-44b4-8ce1-d4b20ac8ae1e.png)

**Comment by Elevator89** [comment]
@MarisKori Да. И в последней версии локализации этих страшных неработающих макросов уже нет.

**Comment by MarisKori** [comment]
Вроде есть.
![image](https://user-images.githubusercontent.com/18173966/130340190-8e29d0b5-fc2e-4f03-949a-e40bf78aaa99.png)

**Comment by MarisKori** [comment]
![image](https://user-images.githubusercontent.com/18173966/132456829-91e3df92-c000-49a2-8909-3b90183073fe.png)
